features/overflow-overlay.yml

```
@@ -0,0 +1,12 @@
+name: "`overflow: overlay`"
+description: "The `overflow: overlay` CSS declaration sets the behavior for when content doesn't fit in an element. It is superceded by `overflow: auto`."
```

Yikes, this one is messy. According to https://caniuse.com/css-overflow-overlay, this has a rather different between versions that BCD doesn't capture at all. I suspect what this needs is new support statements in BCD, showing `partial_implementation` when browsers had the non-standard behavior (i.e., before they aliased to `auto`).

Maybe we thread this needle like this:

```suggestion
description: "The `overflow: overlay` CSS declaration is an alias to `overflow: auto`. However, non-standard historic implementations change the appearance of scrollbars when content doesn't fit in an element."
```

And then keep this whole file in `drafts` until we (I?) can do the work upstream.

(cc: @atopal this is another good example of a "feature" that is interoperable—`overflow: overlay` aliasing to `overflow: auto`—but a bad thing to describe as Baseline)

---

After doing the work on the compat data, I'm now suggesting:

```suggestion
description: "The `overflow: overlay` CSS declaration is an alias to `overflow: auto`. Historically, it caused non-standard behavior, allowing scrollbars to overlay content without taking up layout space."
```